spec/controllers/menus_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe MenusController, type: :controller do
+  describe 'GET #index' do
+    it 'returns success response' do
+      get :index
+      expect(response).to be_ok
```

Comment:
Please, take a look at Pivorak's test structure - https://github.com/pivorakmeetup/pivorak-web-app/tree/development/spec 
Consider applying it here